### PR TITLE
CIP-0024 | Adjust preamble and structure w.r.t CIP-0001

### PR DESCRIPTION
**Pull request by rphair**
Fixes https://github.com/cardano-foundation/CIPs/issues/534.
Completes CIP title update begun in https://github.com/cardano-foundation/CIPs/pull/536.

### CODE REVIEW

File: CIP-0024/README.md [hunk 1]
```diff
@@ -81,20 +81,22 @@ As an example, setting h to 0.1 would mean that the initial number of epochs for
...
+Using this non-centralizing ranking methodology gives a more fair ranking of stake pools based on performance, pledge and saturation which will encourage delegators to choose better pools.
 It will also bring the rankings more in line with the general Cardano principle of increasing decentralization.
 
-## Backward Compatibility
+## Path to Active
+
+### Acceptance Criteria
 
-This proposal does not break backwards compatability because it is an offchain change.
+Confirmation by Daedalus development team that the changes to Cardano's proof-of-stake incentives are considered acceptable and desirable.
```
Comment (KtorZ) [inline CIP-0024/README.md:93]:
```suggestion
- [ ] One or more wallet software implements this new ranking approach.
```

File: CIP-0024/README.md [hunk 2]
```diff
@@ -81,20 +81,22 @@ As an example, setting h to 0.1 would mean that the initial number of epochs for
...
 
-## Implementations
+### Implementation Plan
 
-If someone will show me where the current desirability equation is implemented in the code, I could produce an implementation of this change as a pull request.
+Author has offered to produce an implementation of this change as a Daedalus pull request if shown where the current desirability equation is implemented in the code.
```
Comment (KtorZ) [inline CIP-0024/README.md:97]:
```suggestion
- [ ] Author has offered to produce an implementation of this change as a cardano-wallet / Daedalus pull request if shown where the current desirability equation is implemented in the code.
```

**Review by KtorZ — APPROVED** [review]

**Comment by rphair** [comment]
https://github.com/cardano-foundation/CIPs/pull/538#discussion_r1251800219 and https://github.com/cardano-foundation/CIPs/pull/538#discussion_r1251801460 suggest we are decoupling this CIP from Daedalus.

Note @KtorZ @Ryun1 that https://github.com/cardano-foundation/CIPs/pull/536 puts Daedalus in the CIP Table title, and this PR puts Daedalus into the CIP document title... which reflects the way the CIP is written.  If we agree that this CIP is in fact intended to apply to other wallets besides Daedalus then I'll reverse those changes here.

**Comment by KtorZ** [comment]
I think the CIP is broader than just Daedalus. While it starts from an observation in the Daedalus' ranking, it was also written at a time were there were not as many wallets as today on Cardano. 

Having the CIP framed more as an alternative ranking solution would increase its chance of being adopted as I see very little chance for it being implemented by the Daedalus  / cardano-wallet team after all these years.

**Comment by Ryun1** [comment]
I see that this proposal could be generalized to other wallets, but also the author focuses solely on Daedalus. 

I think we can either reframe this to fit in the more general `wallets` category OR we set to inactive as Deadalus is not an enlisted CIP category.

**Comment by Ryun1** [comment]
> but it focuses on the official Shelley specification

ahh youre right, I had not read it carefully enough, disregard my [last comment](https://github.com/cardano-foundation/CIPs/pull/538#issuecomment-1620029051).

**Comment by rphair** [comment]
good @KtorZ @Ryun1, we're in agreement then ... I'll commit a further change that removes the Daedalus term again from both titles.

**Comment by rphair** [comment]
OK @Ryun1 that's been done a couple days ago... can we have another look? 🤠

**Review by Ryun1 — APPROVED** [review]
Looks good to me, great work @rphair 🚀

**Pull request merged**